rules/manager.go

```diff
@@ -616,13 +638,16 @@ func (g *Group) Eval(ctx context.Context, ts time.Time) {
 			seriesReturned := make(map[string]labels.Labels, len(g.seriesInPreviousEval[i]))
 			defer func() {
 				if err := app.Commit(); err != nil {
+					g.metrics.ingestionFailuresTotal.WithLabelValues(groupKey(g.File(), g.Name())).Add(float64(len(seriesReturned)))
 					level.Warn(g.logger).Log("msg", "Rule sample appending failed", "err", err)
 					return
 				}
 				g.seriesInPreviousEval[i] = seriesReturned
 			}()
 			for _, s := range vector {
+				g.metrics.ingestionSamplesTotal.WithLabelValues(groupKey(g.File(), g.Name())).Inc()
```

Comment:
It'd be better to do these as one increment after the for.

Reply:
Done that :+1: